compute/apiv1/smoke_test.go

```diff
@@ -570,7 +571,16 @@ func TestCapitalLetter(t *testing.T) {
 				Firewall: name,
 			})
 		if err != nil {
-			t.Error(err)
+			// sometimes the resource is not "ready" for deletion, sleep and retry.
+			time.Sleep(60 * time.Second)
```

Comment:
Instead of a single hardcoded sleep I think we should make use of internal.Retry:

https://github.com/googleapis/google-cloud-go/blob/master/internal/retry.go#L30-L32

Reply:
done